packages/vue/src/composables/useAuth.ts

```diff
@@ -23,9 +23,7 @@ const useInternalAuthenticator = () => {
   return useAuthenticatorValue;
 };
 
-export const useAuthenticator = createSharedComposable(
-  useInternalAuthenticator
-);
+export const useAuthenticator = useInternalAuthenticator;
```

Comment:
We no longer wrap the `createSharedComposable` any longer. This has been moved to each individual `.vue` file. This helps fix a problem with reactivity. It also more closely matches the proper use of the `composable` from the documentation.